server/middlewares/validations.js

```diff
@@ -48,6 +49,44 @@ class inputValidator {
     }
     next();
   }
+
+  /**
+   * @description signinValidator
+   * @param {object} request http request object
+   * @param {object} response http response object
+   * @param {object} next callback function
+   * @returns {object} http response object
+   */
+  static editProfile(request, response, next) {
+    const { email, username, password } = request.body;
+    if (username || email || password) {
+      return badHttpResponse(
+        response,
+        400,
+        'Email or username cannot be changed at this time.',
```

Comment:
why not?

Reply:
It was the team's recommendation when I asked them.